apps/docs/app/ui/layout/header/header.css

```diff
@@ -33,7 +33,11 @@
     z-index: var(--hd-z-index-1000);
 }
 
-@media screen and (width >= 37.5rem) {
+.hd-header .hd-layout__fluid {
```

Comment:
It is also necessary to make the change in the sub-header so that the elements remain aligned.

<img width="684" alt="image" src="https://github.com/user-attachments/assets/0b9264a5-0451-4a70-9b4b-6d5e9a52b501">